wiki/en/Installation-for-iOS.md

```diff
@@ -20,7 +20,9 @@ If you have don't own a PC, we suggest you to buy a [Raspberry Pi](https://www.r
 
 * To install Jamulus on your iOS device, you need a (free or paid) Apple Developer account and preferably a computer.
 * If you don't pay for an Apple developer account, Jamulus will only run for at most 7 days before you need to re-install it.
-* Jamulus on iOS works quite well on new devices if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. To connect these to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered) USB-hub.
+* Jamulus on iOS works quite well if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. If you own an older device which has a Lightning port and not USB-C, you can use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered) USB-hub.
+* If you must use Jamulus on a wireless network, you should prefer a cellular (5G) connection over WiFi.
```

Comment:
Is this iOS-specific?

Reply:
I don't know about android but from experience with a 5G router + Ethernet + Linux, cellular quality is much better compared to WiFi. 
I'd say it's not OS specific.

Reply:
It's probably better left off this page, or at least make it a reference to the general guidance.  (Maybe somewhere like hints and tips would be better).

Reply:
I think it's mainly a mobile thing. Thus it's here (computers with 5G exist, but are rare)

Reply:
Maybe put it on the FAQ page if it isn't already? I wouldn't like to maintain a separate mobile page. That adds too much clutter.

Reply:
We have two pages.

I suggested one page in their place.

That's less "clutter", right?

Reply:
I understood that differently: 1 page per OS and then one page for mobile.

I think having a page per OS is sensible as installation works differently. We can use imports to show text on multiple pages. @gilgongo did that in the past, I think.

So this import would then discuss about mobile.

Reply:
Yes, duplication of content is the main problem to avoid. So we have used includes in the past for sharing the same info between pages (if we think it may need changing in the future). 

Pages per OS seems OK to me, since most people will want to read stuff that presents itself for their OS I think.